boards/arm/disco_l475_iot1/disco_l475_iot1_defconfig

```diff
@@ -48,11 +48,13 @@ CONFIG_STM32_ARM_MPU_ENABLE=y
 
 #enable I2C
 CONFIG_I2C=y
-CONFIG_I2C_STM32_V2=y
 
-#configure HTS221 sensor
-CONFIG_HTS221_I2C_MASTER_DEV_NAME="I2C_2"
-CONFIG_HTS221_TRIGGER_NONE=y
+#Enable sensors
```

Comment:
Those would need to be enabled on the application side, they would not have any effect here. So probably it makes sense to document them as possible options and provide a sample that uses them instead of setting them to n here.